vendor/github.com/google/cel-go/ext/bindings.go

```diff
@@ -41,35 +51,120 @@ import (
 //	[d, e, f].exists(elem, elem in valid_values))
 //
 // Local bindings are not guaranteed to be evaluated before use.
-func Bindings() cel.EnvOption {
-	return cel.Lib(celBindings{})
+func Bindings(options ...BindingsOption) cel.EnvOption {
+	b := &celBindings{version: math.MaxUint32}
```

Comment:
looks like dra is using this library without specifying a version, so is instantly exposed to new stuff in version 1

```
				// https://pkg.go.dev/github.com/google/cel-go/ext#Bindings
				//
				// This is useful to simplify attribute lookups because the
				// domain only needs to be given once:
				//
				//    cel.bind(dra, device.attributes["dra.example.com"], dra.oneBool && dra.anotherBool)
				ext.Bindings(),
```

unversioned cel libraries silently adding a second version and defaulting to enable latest is likely to bite us... we can pass in a version if we catch that it became versioned in a bump, but that requires *super* careful review... let's think about how to put something in place so we notice if the library constructor signature changes to become versioned. Could be as simple as:

```
type unversionedLib func() cel.EnvOption
...
			unversionedLib(library.URLs())(),
			unversionedLib(library.Regex())(),
			unversionedLib(library.Lists())(),
```

Reply:
Good call. We need this. I'll add it.

Reply:
Fixed in latest commit.